wavelength-plugins/friends.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * FC Friends
+ * By WGC
+ */
```

Comment:
I would place an empty line after this comment, just because it'll match the rest of the files in the repo.